app/controllers/api/v1/public_api/talents_controller.rb

```diff
@@ -34,10 +34,11 @@ def index
   end
 
   def recommended
-    per_page = 3
-
     subscribed_users = Subscription.where(user_id: current_user.id)
```

Comment:
deleted line being used below.

Also

```suggestion
    subscribed_users = Subscription.where(user: current_user)
```